test/unit/classes/classes.spec.ts

```diff
@@ -706,118 +625,109 @@ test("Class Method Runtime Override", () => {
         inst.method = () => {
             return 8;
         }
-        return inst.method();`
-    );
-
-    expect(result).toBe(8);
+        return inst.method();
+    `.expectToMatchJsResult();
 });
 
 test("Exported class super call", () => {
-    const code = `
-        export class Foo {
-            prop: string;
-            constructor(prop: string) { this.prop = prop; }
-        }
-        export class Bar extends Foo {
-            constructor() {
-                super("bar");
-            }
-        }
-        export const baz = (new Bar()).prop;
-    `;
-    expect(util.transpileExecuteAndReturnExport(code, "baz")).toBe("bar");
+    util.testExpression`
+        (new Bar()).prop;
+    `
+        .setTsHeader(
```

Comment:
I'm probably going to remove `setTsHeader` in the future, so it's better to test it with `testModule`